### PR TITLE
Removed EBI layer on loading-related pipelines

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
Ticket [ENSCOMPARASW-2486](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-2486) (part 1/7).

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/CITest/BuildCITestMasterDatabase_conf.pm [hunk 1]
```diff
@@ -51,7 +51,8 @@ Bio::EnsEMBL::Compara::PipeConfig::EBI::CITest::BuildCITestMasterDatabase_conf
 
 =head1 SYNOPSIS
 
-    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::CITest::BuildCITestMasterDatabase_conf -dst_host <host> -dst_port <port>
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::CITest::BuildCITestMasterDatabase_conf -host mysql-ens-compara-prod-X -port XXXX \
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/CITest/BuildCITestMasterDatabase_conf.pm]:
```suggestion
    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::CITest::BuildCITestMasterDatabase_conf -host mysql-ens-compara-prod-X -port XXXX \
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm [hunk 2]
```diff
@@ -87,14 +95,22 @@ sub default_options {
         'uniprot_rel_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/reldate.txt',
         'uniprot_ftp_url' => 'ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/uniprot_#uniprot_source#_#tax_div#.dat.gz',
 
-        #'blast_params'    => '', # By default C++ binary has composition stats on and -seg masking off
+        'blast_params'    => '', # By default C++ binary has composition stats on and -seg masking off
+
+        # data directories:
+        'warehouse_dir' => '/nfs/production/panda/ensembl/warehouse/compara/production/'.$self->o('ensembl_release').'/Families_'.$self->o('rel_with_suffix'),
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm]: The path to the compara warehouse feels like something that should be in the ENV, IMO

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm [hunk 3]
```diff
@@ -64,29 +64,10 @@ sub default_options {
...
-        # Change this one to allow multiple runs
-        #'rel_suffix'            => 'b',
         'division'   => 'vertebrates',
-        # 'collection' => $self->o('division'),
-        'collection' => 'vertebrates',
-
-        # names of species we don't want to reuse this time
-        #'do_not_reuse_list'     => [ 'homo_sapiens', 'mus_musculus', 'rattus_norvegicus', 'mus_spretus_spreteij', 'danio_rerio', 'sus_scrofa' ],
-        'do_not_reuse_list'     => [ ],
+        'collection' => $self->o('division'),
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm]: Could be in base class

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm [hunk 4]
```diff
@@ -64,29 +64,10 @@ sub default_options {
...
-        # It is very important to check that this value is current (commented out to make it obligatory to specify)
-        # Change this one to allow multiple runs
-        #'rel_suffix'            => 'b',
         'division'   => 'vertebrates',
-        # 'collection' => $self->o('division'),
-        'collection' => 'vertebrates',
-
-        # names of species we don't want to reuse this time
-        #'do_not_reuse_list'     => [ 'homo_sapiens', 'mus_musculus', 'rattus_norvegicus', 'mus_spretus_spreteij', 'danio_rerio', 'sus_scrofa' ],
-        'do_not_reuse_list'     => [ ],
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm]: I think I would have kept this. I know you were removing anything that is set the same as the base class, but this is one that is changed quite regularly (relative to other settings anyway)

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/LoadMembers_conf.pm [hunk 5]
```diff
@@ -60,38 +64,14 @@ sub default_options {
...
-        # It is very important to check that this value is current (commented out to make it obligatory to specify)
-        # Change this one to allow multiple runs
-        #'rel_suffix' => 'b',
-
         'division'    => 'plants',
         'collection'  => $self->o('division'),
 
-        # names of species we don't want to reuse this time
-        #'do_not_reuse_list'     => [ 'homo_sapiens', 'mus_musculus', 'rattus_norvegicus', 'mus_spretus_spreteij', 'danio_rerio', 'sus_scrofa' ],
-        'do_not_reuse_list'     => [ ],
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/LoadMembers_conf.pm]: Same as vertebrates - prob would have kept this one, personally

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm (hunk 4 above)
Comment (JAlvarezJarreta) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm]: My approach is that you can always change it from the command line when creating the pipeline, but I can put it back the commented part if you feel it is better.

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm (hunk 2 above)
Comment (muffato) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/Families_conf.pm]: Well you're doing a lot more than just "removing the EBI layer" anyway ;)
I think it's a good point. ENV already has `pipeline_dir` and would welcome a `warehouse_dir` built on the same model that would then be available to all pipelines.

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/Families_conf.pm [hunk 6]
```diff
@@ -73,37 +74,6 @@ sub default_options {
         %{ $self->SUPER::default_options },
 
         'division' => 'vertebrates',
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/Families_conf.pm]: If that's the only thing left, we may not need this PipeConfig at all ?

**Pull request merged**